app/Filament/Admin/Resources/NodeResource/Pages/CreateNode.php

```diff
@@ -232,6 +232,8 @@ public function form(Forms\Form $form): Forms\Form
                             TextInput::make('upload_size')
                                 ->label('Upload Limit')
                                 ->helperText('Enter the maximum size of files that can be uploaded through the web-based file manager.')
+                                ->hintIcon('tabler-question-mark')
+                                ->hintIconTooltip('Make sure you change client_max_body_size, client_body_timeout, upload_max_filesize = 100M & post_max_size=100M in your webserver config')
```

Comment:
Let's keep the tooltip simple. Most people woudn't know how and where to change these things anyways.

```suggestion
                                ->hintIconTooltip('Make sure your webserver supports file uploads of this size!')
```

Reply:
Agreed